### PR TITLE
Add staking debugging tools to localnet deploy script

### PR DESCRIPTION
**Pull request by Daniel-VDM**
* `./test/deploy.sh` will now create validators using the hmy CLI. This set-up also allows us to easily add more staking tests/tools in the future.

> *Dependencies*: Python3 + [requests package](https://pypi.org/project/requests/) + [pyhmy package](https://pypi.org/project/pyhmy/) + built hmy CLI binary in `harmony-one/go-sdk` + latest version of `harmony-one/harmony-ops` cloned. (Packages are checked and installed if needed in `./test/deploy.sh`)

**Do not merge this to master until [this PR](https://github.com/harmony-one/harmony-ops/pull/235) is merged to master**

### CODE REVIEW

**Comment by LeoHChen** [comment]
why do we need a separate test script? can it somehow merge with the existing test script?

**Comment by fxfactorial** [comment]
because not ready to be mainline production yet but need it for development atm to unblock frontend explorer work

**Comment by fxfactorial** [comment]
Actually nvm this should be fine because just one of five kinds of staking txs that works

**Comment by Daniel-VDM** [comment]
Jenkins, test this please

File: test/deploy.sh [hunk 1]
```diff
@@ -59,6 +59,17 @@ function cleanup_and_result() {
    [ -e $RESULT_FILE ] && cat $RESULT_FILE
 }
 
+function debug_staking() {
+   hmy_one_dir="$(go env GOPATH)/src/github.com/harmony-one"
+   hmy_bin="${hmy_one_dir}/go-sdk/hmy"
+   keystore="${hmy_one_dir}/harmony-ops/test-automation/api-tests/LocalnetValidatorKeys"
+   python3 -m pip install pyhmy
```
Comment (LeoHChen) [inline test/deploy.sh]: do you have to install it every time? can you check if it is installed?

Comment (Daniel-VDM) [reply test/deploy.sh]: that command only installs if it is not installed

**Comment by LeoHChen** [comment]
@Daniel-VDM , please use your own fork to submit PR to main repo.  Don't create a branch in the main harmony repo.  Thanks.

**Pull request merged**